app/src/main/java/com/globant/practice/presentation/model/WebClientState.java

```diff
@@ -9,6 +9,8 @@
     private String htmlUrl;
     private String detailType;
     private boolean loading;
+    private String error;
+    private boolean errorShowing;
```

Comment:
It's not clear for me the purpose of this attribute. Can you explain it and tell under what circumstances is useful?